### PR TITLE
Fix marker clustering

### PR DESCRIPTION
**Pull request by sarveshpro**
Fix #933

Changes in this pull request:
- fix marker clustering on my trees map

- [x] Add on hover details

### CODE REVIEW

**Comment by sagararyal** [comment]
Screen Shot 2021-03-20 at 2 01 46 PM

**Comment by harshvitra** [comment]
Can you check the white space

Screenshot 2021-03-24 at 7 43 37 PM

File: index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  let taskTime = 0;
+  arr.map((el) => {
+    taskTime += el[1] - el[0];
+  });
+
+  return totalTime - taskTime;
+}
+
+console.log(breakTime(arr, tasks));
```
Comment (norbertschuler) [inline index.js:32]: I assume this file should not have been committed - please remove it.

**Review by norbertschuler — APPROVED** [review]
If the removal of the tree list overview to the left of the map is really intended, it seems to work for me. I just wonder where buttons to edit or delete a contribution will be placed later on.

File: index.js (hunk 1 above)
Comment (sarveshpro) [reply index.js:32]: Yes my bad

**Review by harshvitra — CHANGES_REQUESTED** [review]

**Comment by sarveshpro** [comment]
> If the removal of the tree list overview to the left of the map is really intended, it seems to work for me. I just wonder where buttons to edit or delete a contribution will be placed later on.

We have to readd it with a better design

**Comment by harshvitra** [comment]
- [ ] Border radius on map for public profile

**Review by harshvitra — APPROVED** [review]

**Comment by sagararyal** [comment]
Do not merge

**Pull request merged**